sympy/physics/control/tests/test_control_plots.py

```diff
@@ -92,33 +92,35 @@ def test_errors():
     raises(ValueError, lambda: ramp_response_plot(tf1, slope=-0.1))
 
 
+@XFAIL
```

Comment:
Why is this XFAIL:ed? Seems like if anything an XFAIL-should be removed.

Also, why is this skipped based on matplotlib? Seems like this is only numerical methods?

Is numpy requited to generate the numerical data? If not, one should maybe create a local version of `allclose`? There is an `isclose` in math that one can iterate over.

Reply:
> Why is this XFAIL:ed? Seems like if anything an XFAIL-should be removed.

**The test runs perfectly fine on my computer.** However, the issue is that the order of roots computed by NumPy (`np.roots()` is used in the pole-zero plot) seems to vary, either due to updated code in newer releases of NumPy or due to different CPU architecture (I'm not sure). The order of complex roots returned by the `pole_zero_numerical_data()` earlier was different from now. It can lead to exceptions when running the test suite since we make a one-to-one comparison for each array element.

Is there any way to ensure that the roots are returned in a fixed order?

I added this XFAIL marker because I thought this test might fail for the users for which the order of computed roots is different. I can remove it if someone can confirm that the order of roots returned is same for every user.

> Is numpy requited to generate the numerical data? If not, one should maybe create a local version of `allclose`? There is an `isclose` in math that one can iterate over.

This seems to be an excellent option. Only the `pole_zero_numerical_data()` function requires NumPy for calculation. All other functions in the module utilize numeric computation native to sympy.